homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -210,6 +214,18 @@ def setup(hass: HomeAssistant, base_config):
         adapter = CecAdapter(name=display_name[:12], activate_source=False)
     hdmi_network = HDMINetwork(adapter, loop=loop)
 
+    def _adapter_watchdog(now=None):
```

Comment:
This isn't marked with `@callback` so will run in the executor thread pool. But we're using async api inside. This is not correct.